engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/KafkaAvroSourceFactory.scala

```diff
@@ -47,20 +50,33 @@ class KafkaTypedAvroSourceFactory[T: TypeInformation](schemaRegistryProvider: Sc
              )
              @NotBlank
              topic: String,
-             // json or avro schema on this level?
              @ParamName("schema")
              @NotBlank
-             avroSchema: String): Source[T] with TestDataGenerator = {
+             stringSchema: String): Source[T] with TestDataGenerator =
+    createKafkaAvroSource(
+      processMetaData,
+      topic,
+      new KafkaAvroFixedSchemaProvider(topic, stringSchema, kafkaConfig, formatKey, useSpecificAvroReader)
```

Comment:
Maybe better will be `FixedKafkaAvroSchemaProvider` - below you have name `FixedKafkaAvroSourceFactory` in this convension.

Reply:
Changed.